images/orbit-controller/src/orbit_controller/operators/userspace_operator.py

```diff
@@ -95,6 +96,41 @@ def _install_helm_chart(
     return install_status
 
 
+def _create_user_efs_endpoint(user: str, team_name: str, team_efsid: str, env: str) -> Dict[str, Any]:
+    efs = boto3.client("efs")
+
+    return cast(
+        Dict[str, str],
+        efs.create_access_point(
+            FileSystemId=team_efsid,
+            PosixUser={"Uid": 1000, "Gid": 100},
+            RootDirectory={
+                "Path": f"/{team_name}/private/{user}",
+                "CreationInfo": {"OwnerUid": 1000, "OwnerGid": 100, "Permissions": "770"},
+            },
+            Tags=[{"Key": "TeamSpace", "Value": team_name}, {"Key": "Env", "Value": env}],
+        ),
+    )
+
+
+def _delete_user_efs_endpoint(user_name: str, user_namespace: str, logger: kopf.Logger, meta: kopf.Meta) -> None:
+    efs = boto3.client("efs")
+
+    logger.info(f"Fetching the EFS access point in the namespace {user_namespace} for user {user_name}")
+
+    efs_access_point_id = meta.get("labels", {}).get("userEfsApId", None)
+
+    logger.info(f"Deleting the EFS access point {efs_access_point_id} for user {user_name}")
+
+    try:
+        efs.delete_access_point(AccessPointId=efs_access_point_id)
+        logger.info(f"Access point {efs_access_point_id} deleted")
+    except efs.exceptions.AccessPointNotFound:
+        logger.error(f"Access point not found: {efs_access_point_id}")
```

Comment:
should this and the next one be logger.warn since it's not a fatal error?

Reply:
Agreed!